core/bind/core_bind.cpp

```diff
@@ -1827,6 +1828,11 @@ void _File::_bind_methods() {
 	BIND_CONSTANT(WRITE);
 	BIND_CONSTANT(READ_WRITE);
 	BIND_CONSTANT(WRITE_READ);
+
+	BIND_CONSTANT(COMPRESSION_FASTLZ);
+	BIND_CONSTANT(COMPRESSION_DEFLATE);
+	BIND_CONSTANT(COMPRESSION_ZSTD);
```

Comment:
There's no zstd in the 2.1 branch, so this should not be added (in this patch it also isn't used).